src/Microsoft.AspNetCore.Http/DefaultHttpContext.cs

```diff
@@ -44,15 +46,26 @@ public DefaultHttpContext()
         }
 
         public DefaultHttpContext(IFeatureCollection features)
+            : this(new FeatureCollection(), FormOptions.Default)
```

Comment:
Shouldn’t we be passing the feature collection and not a new one?

Reply:
Done